recipes-graphics/libwebrtc/libwebrtc_125.6422.06.bb

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) 2024 Joel Winarske. All rights reserved.
 #
 
-SUMMARY = "libbrtc"
+SUMMARY = "libwebrtc"
```

Comment:
should it also be in separate PR?

Reply:
this would fall under agnostic PR, so not related to required master changes